index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Welcome Page</title>
+</head>
+<body>
```

Comment:
Add navigation bar

Reply:
added in d0cbfd3b4b42255815c95cb7ca3b90fb8bbb7044

Reply:
Wow. Wonderful